lib/biran/config_defaults.rb

```diff
@@ -11,7 +11,6 @@ def app_defaults_init
           shared_dir: configuration.shared_dir,
           base_dir: configuration.base_dir,
           use_capistrano: configuration.use_capistrano,
-          files_to_generate: configuration.files_to_generate,
```

Comment:
This removal means the merge is clean from the supplied config files. If there is no value for this list in app_config.yml, we use the `configureation.files_to_generate values` when needed here - https://github.com/amco/biran/blob/master/lib/biran/configurinator.rb#L38